src/chef-server-bootstrap/bin/bootstrap-platform

```diff
@@ -85,7 +93,26 @@ class BootstrapPlatform
     ##############################
     # create superuser container #
     ##############################
-    create_global_containers(superuser_authz_id, PLACEHOLDER_GLOBAL_ORG_ID, db, config['bifrost_host'], config['bifrost_port'])
+    global_container_authz_ids = create_global_containers(superuser_authz_id, PLACEHOLDER_GLOBAL_ORG_ID, db, bifrost_host, bifrost_port)
+
+    #####################################
+    # create server-admins global group #
+    #####################################
+    create_server_admins_global_group(db, superuser_authz_id, PLACEHOLDER_GLOBAL_ORG_ID, server_admins_authz_id)
+    # grant this object create and read on the users container
+    grant_authz_object_permission("create", "groups", "containers", global_container_authz_ids['users'], server_admins_authz_id, superuser_authz_id, bifrost_host, bifrost_port)
+    grant_authz_object_permission("read",   "groups", "containers", global_container_authz_ids['users'], server_admins_authz_id, superuser_authz_id, bifrost_host, bifrost_port)
+    grant_authz_object_permission("update", "groups", "containers", global_container_authz_ids['users'], server_admins_authz_id, superuser_authz_id, bifrost_host, bifrost_port)
+    grant_authz_object_permission("delete", "groups", "containers", global_container_authz_ids['users'], server_admins_authz_id, superuser_authz_id, bifrost_host, bifrost_port)
+
+    ###########################################
+    # put pivotal in site-admins global group #
+    ###########################################
+    grant_authz_object_permission("create", "actors", "groups", server_admins_authz_id, superuser_authz_id, bifrost_superuser_id, bifrost_host, bifrost_port)
+    grant_authz_object_permission("read",   "actors", "groups", server_admins_authz_id, superuser_authz_id, bifrost_superuser_id, bifrost_host, bifrost_port)
+    grant_authz_object_permission("update", "actors", "groups", server_admins_authz_id, superuser_authz_id, bifrost_superuser_id, bifrost_host, bifrost_port)
+    grant_authz_object_permission("delete", "actors", "groups", server_admins_authz_id, superuser_authz_id, bifrost_superuser_id, bifrost_host, bifrost_port)
```

Comment:
Yeah delete on the global-group itself is probably unnecessary / a footgun. Can remove.